docs/commands.md

```diff
@@ -6579,8 +6579,10 @@ hal config notification slack edit [parameters]
 ```
 
 #### Parameters
+ * `--base-url`: Slack endpoint.
```

Comment:
Can you indicate that this parameter is optional?

Reply:
Is the comment good like this? I accept other suggestions :)